docs/source/delta-row-tracking.md

```diff
@@ -0,0 +1,113 @@
+---
+description: Learn how <Delta> row tracking allows tracking how rows change across table versions.
+orphan: 1
+---
+
+# Use row tracking for Delta tables
+
+Row tracking allows <Delta> to track row-level lineage in a <Delta> table. When enabled on a <Delta> table, row tracking adds two new metadata fields to the table:
+
+- **Row IDs** provide rows with an identifier that is unique within the table. A row keeps the same ID whenever it is modified using a `MERGE` or `UPDATE` statement.
+
+- **Row commit versions** record the last version of the table in which the row was modified. A row is assigned a new version whenever it is modified using a `MERGE` or `UPDATE` statement.
```

Comment:
It could be worth quickly highlighting here that these can be accessed via `_metadata.row_id` and _metadata.row_commit_version` even if there's more detail about it below

Reply:
I'm trying to keep this first section a little bit more high-level.